keps/976-plain-pods/kep.yaml

```
@@ -0,0 +1,31 @@
+title: Plain Pods
+kep-number: 976
+authors:
+  - "@alculquicondor"
+status: provisional
+creation-date: 2023-07-31
+reviewers:
+  - "@trasc"
+approvers:
+  - "@ahg-g"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v0.5"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v0.5"
```

```suggestion
  beta: "v0.5"
```

Not sure if you all have alpha in Kueue.

---

We do, but in this case, we won't have a feature gate.